### PR TITLE
heat_maps: add resolution and requested_date_time

### PR DESCRIPTION
**Pull request by ZoeBrunet**

### CODE REVIEW

File: response.proto [hunk 1]
```diff
@@ -384,17 +384,19 @@ message Link {
...
-    optional int32 max_duration   = 2;
-    optional int32 min_duration   = 3;
-    optional PtObject origin      = 4;
-    optional PtObject destination = 5;
+    optional string geojson             = 1;
+    optional int32 max_duration         = 2;
+    optional int32 min_duration         = 3;
+    optional PtObject origin            = 4;
+    optional PtObject destination       = 5;
+    optional uint64 requested_date_time = 6;
```
Comment (pbougue) [inline response.proto]: `requested_datetime` (no _)

File: response.proto [hunk 2]
```diff
@@ -384,17 +384,19 @@ message Link {
...
 }
 
 message HeatMap {
-    optional PtObject origin        = 2;
-    optional PtObject destination   = 3;
-    optional string heat_matrix     = 1;
+    optional string heat_matrix     	= 1;
+    optional PtObject origin            = 2;
+    optional PtObject destination       = 3;
+    optional uint64 requested_date_time = 4;
```
Comment (pbougue) [inline response.proto]: same

File: response.proto (hunk 1 above)
Comment (ZoeBrunet) [inline response.proto]: It is requested_date_time in journeys

Comment (pbougue) [inline response.proto]: then we let it as is, yes.
ok for me

**Pull request merged**